lib/kramdown/converter/math_engine/sskatex/details.rb

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
```

Comment:
So that you can `require` it from `sskatex.rb`. If that fails (typically because of unavailability of `execjs`), `sskatex.rb` defines only `SsKaTeX::AVAILABLE = false`).

Reply:
Have a look at the other math engine or syntax highlighter implementations. They have the same problem but don't include an extra file.